File: server/DTOS/OrderDTOs/OrderReadDTO.cs
using static Order;

public class OrderReadDTO
{
    public int Id { get; set; }
    public int UserId { get; set; }
    public DateTime CreatedAt { get; set; }
    public decimal TotalPrice { get; set; }

    public List<OrderItemReadDTO>? OrderItems { get; set; }
    public OrderStatus Status { get; set; }

}

File: admin-panel/src/types/OrderTypes/OrderReadDTO.ts
import type { OrderItemReadDTO } from "./OrderItemReadDTO";

export interface OrderReadDTO {
  Id:number;
  UserId: number;
  CreatedAt: string;
  TotalPrice: number;
  Status: OrderStatus; // enum tipi
  OrderItems: OrderItemReadDTO[];
  ImgUrl:string
  ProductName:string
}

export type OrderStatus = 
  | "Pending"
  | "Processing"
  | "Shipped"
  | "Delivered"
  | "Cancelled";
